src/components/zen-tab/test/zen-tab.spec.tsx

```diff
@@ -10,4 +11,36 @@ describe('zen-tab', () => {
 
     expect(page.root.shadowRoot).toBeTruthy();
   });
+
+  it('should display a badge', async () => {
+    const page = await newSpecPage({
+      components: [ZenTab],
+      html: `<zen-tab badge="123">Tab 1</zen-tab>`,
+    });
+
+    expect(page.root.shadowRoot.querySelector('.badge')).toBeTruthy();
+  });
+
+  it('should be disabled', async () => {
+    const page = await newSpecPage({
+      components: [ZenTab],
+      html: `<zen-tab disabled>Tab 1</zen-tab>`,
+    });
+
+    expect(page.root.hasAttribute('disabled')).toBeTruthy();
```

Comment:
```suggestion
    expect(page.root.hasAttribute('disabled')).toBe(true);
```

This returns boolean, so we can compare it to `true` directly.

Reply:
Done